hazelcast-spring/src/main/resources/hazelcast-spring-3.5.xsd

```diff
@@ -1977,5 +1977,6 @@
     <xs:complexType name="wan-replication-ref">
         <xs:attribute name="name" use="required" type="xs:string"/>
         <xs:attribute name="merge-policy" use="required" type="xs:string"/>
+        <xs:attribute name="republish-enabled" use="optional" type="xs:boolean" default="false"/>
```

Comment:
`republishing-enabled` sounds more correct to me... even `enable-republishing`...